server/src/main/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimGroupProvisioning.java

```diff
@@ -220,9 +229,24 @@ public ScimGroup retrieve(String id, final String zoneId) throws ScimResourceNot
         }
     }
 
+    @SuppressWarnings("java:S1874")
```

Comment:
Ok, but this is a old discussion old. Must of the S1874 warnings from from deprecated code which we cause, e.g. IdentityZoneHoler or https://github.com/cloudfoundry/uaa/milestone/38 but this annotation simply cleanup  
our own deprecations.....  why is IdentityZoneHoler deprecated but nobody cares about it or removes it

Reply:
IMO, we should suppress the warning when we believe the warning is incorrect. We shouldn't suppress the warnings because we don't like the noise, and because we don't want to fix the underlying issue. 

Here, a real issue is reported by Sonar. We shouldn't hide it.

I am not saying that we need to fix the deprecation problem right now. But IMO we need to remove the tags, and allow Sonar to show that there is a problem.

Reply:
removed